docker-compose.yml

```diff
@@ -203,6 +203,18 @@ services:
     volumes:
       - './bin/wait-for:/unleash/wait-for:z'
       - './.unleash:/.unleash:z'
+  artemis:
```

Comment:
Since this service is only to be used when UMB_ENABLED is true, it should be considered an optional service and hence moved to `config/artemis`. 
This is because the docker-compose.yml should have the minimal as possible services in the default state.